templates/terraform/k8s-master/variables.tf

```diff
@@ -26,3 +26,8 @@ variable vault_path {
   type = string
   description = "Path in Vault where secrets for the master should be stored."
 }
+
+variable service_account_email {
+  type = string
+  description = "The email address of the service account."
+}
```

Comment:
I don't think this arg is needed for the k8s master, since it's not used anywhere.

Reply:
I originally didn't have this here but I was getting this error:
`An argument named "service_account_email" is not expected here.`
in regards to the line I had added to  templates/terraform/processing-project/k8s-master.tf (line 17)

Reply:
Yes, once the input is gone you won't need to pass it at the call-site.

Reply:
Gotcha, removing all service account inputs from k8s-master modules.
Only passing the email for the k8s-node-pool modules.